ansible/roles/operator-pipeline/templates/openshift/pipelines/community-hosted-pipeline.yml

```diff
@@ -409,6 +428,30 @@ spec:
         - name: github_token_secret_key
           value: "$(params.github_token_secret_key)"
 
+    - name: skip-prow-tests
```

Comment:
This can stay here for now, but it will be most likely removed when adding a preflight-trigger to the pipeline.